### PR TITLE
Probes values support

### PR DESCRIPTION
**Pull request by viliger18**
Need it in my environment and rather not store a local copy of the chart.
Nothing fancy just support for liveness and readiness values in values.yaml

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -177,3 +177,25 @@ clusterRole:
...
+  initialDelaySeconds: 10
+  periodSeconds: 10
+  timeoutSeconds: 1
+  successThreshold: 1
+  failureThreshold: 3
+
+readinessProbe:
+  enabled: true
+  httpGet:
+    path: /ready
```
Comment (ralgozino) [inline chart/values.yaml:195]: we don't have a /ready endpoint. In the plain manifests we are using `/health` for both probes

```suggestion
    path: /health
```

File: chart/values.yaml [hunk 2]
```diff
@@ -177,3 +177,25 @@ clusterRole:
   # The name of a cluster role to bind to; if not set and create is
   # true, a name based on fullname is generated
   name: "gatekeeper-policy-manager-crd-view"
+# Health and liveness probe configurations
+livenessProbe:
+  enabled: true
+  httpGet:
+    path: /health
+    port: 8080
```
Comment (ralgozino) [inline chart/values.yaml:185]: is better to use the port name
```suggestion
    port: http
```

File: chart/values.yaml [hunk 3]
```diff
@@ -177,3 +177,25 @@ clusterRole:
...
+  periodSeconds: 10
+  timeoutSeconds: 1
+  successThreshold: 1
+  failureThreshold: 3
+
+readinessProbe:
+  enabled: true
+  httpGet:
+    path: /ready
+    port: 8080
```
Comment (ralgozino) [inline chart/values.yaml:196]:
```suggestion
    port: http
```

**Review by ralgozino — CHANGES_REQUESTED** [review]
Hello!

Thanks for the PR. I don't think I understand though. We already had the probes in the chart as you can see in the diff, but they were hardcoded and not configurable in the `values`.

What is it that you need in your environment?

**Comment by viliger18** [comment]
Mainly our problem was that the timeout and and failure threshold were too low, initially just saved the chart and deployed with different values but saw this project is still maintained so prefer to just get the chart and change the hardcoded values of the probes

**Review by ralgozino — APPROVED** [review]

**Comment by ralgozino** [comment]
Hi @viliger18 

We've released a new version of the chart with your added fields:
https://github.com/sighupio/gatekeeper-policy-manager/releases/tag/gatekeeper-policy-manager-0.13.0

Thank you for the collaboration!

**Pull request merged**